sp-sim/src/ereport.rs

```diff
@@ -158,6 +149,30 @@ impl EreportState {
 
             (&Default::default(), req.start_ena)
         };
+
+        let mut respondant_ereports = self
+            .ereports
+            .iter()
+            .filter(|ereport| ereport.ena >= start_ena)
+            .take(req.limit as usize)
+            .peekable();
+        let start_ena = respondant_ereports
+            .peek()
+            .map(|ereport| ereport.ena)
+            .unwrap_or(Ena(start_ena.0 + 1));
```

Comment:
If we're sending no ereports because start_ena is not yet used, this may cause the requester to skip the report with that ENA once assigned. Probably better to send zero here.